adafruit_dash_display.py

```diff
@@ -125,18 +125,23 @@ def last_val(self, value: str):
 
 
 class Hub:  # pylint: disable=too-many-instance-attributes
-    """Object that lets you make an IOT dashboard"""
+    """
+    Object that lets you make an IOT dashboard
+
+    :param display: The display object.
+    :param io: The io object.
+    :param nav: The navigation buttons.
+    """
 
-    # pylint: disable=invalid-name
     def __init__(
         self,
         display: displayio.Display,
-        io: IO_MQTT,
+        io_object: IO_MQTT,
```

Comment:
How about calling this `io_mqtt` instead, since this can only be an `IO_MQTT` object?
```suggestion
        io_mqtt: IO_MQTT,
```
... and change all the other references.

Reply:
This makes a lot of sense. Thank you for the suggestion! Will do!